org.eclipse.lsp4e.jdt/META-INF/MANIFEST.MF

```diff
@@ -14,4 +17,9 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.lsp4e;bundle-version="0.18.13",
  org.eclipse.jdt.core;bundle-version="3.20.0",
  org.eclipse.jdt.ui;bundle-version="3.20.0",
```

Comment:
The version should be bumped to the 1st version that introduces `org.eclipse.jdt.ui.text.java.ISemanticTokensProvider`